ui/webui/src/helpers/conf.js

```diff
@@ -79,3 +80,7 @@ export const readConf = () => {
             .then(parseIni)
             .finally(confFile.close);
 };
+
+export const runningLocally = () => {
+    return window.origin.startsWith(`http://${LOCAL_IP_ADDRESS}`);
```

Comment:
I wonder if this could be https as well at some point when running locally? I guess we can fix that when/if it happens, though.